Client/admin_page.html

```diff
@@ -90,7 +90,14 @@
         <th>ID</th>
         <th>Name</th>
         <th>Image</th>
+        <th></th>
       </tr>
+    <tr>
+        <td><input type="number" name="id"/></td>
```

Comment:
Delete _id_ input
You don't need POST category's id, it is generated automatically and is returned in response. The sample in categories.test.http sets category`s id for convenience of testing.